docs/source/software/quick_start_guide.rst

```diff
@@ -2,58 +2,78 @@
-    * ./install_geographiclib_datasets.sh
-    * Terminator (see comments in https://askubuntu.com/questions/1006448/how-to-install-the-latest-version-of-terminator)
-    * Visual Studio Code (https://code.visualstudio.com/download)
-    * QGroundControl (https://docs.qgroundcontrol.com/en/getting_started/download_and_install.html)
-    * sudo apt install git
-    * sudo apt install openssh-server
-    * sudo apt install screen
-    * pip install --upgrade pip
-    * pip install scipy-stack
-
-8. For python interaction with the HAT, install the adafruit libraries using the instructions in https://forum.up-community.org/discussion/3569/solved-getting-an-adafruit-servo-hat-to-work-on-an-up-board#latest
-9. Connect telem2 to the rx/tx/gnd on the UpBoard. DO NOT CONNECT THE 5V pin.
-10. Setup px4 side: https://dev.px4.io/master/en/companion_computer/pixhawk_companion.html
-11. Setup upboard side: stty -F /dev/ttyS4 921600 cs8 -cstopb -parenb
+.. TODO: Generate quickstart script and usb sticks
 
 Ground Station Setup
---------------------
-This assumes you going to run Ubuntu on the ground station. Well, you should.
\ No newline at end of file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Setup for motion capture (relay.local and mocap.local) can be found at https://stanfordflightroom.github.io/documentation.
+In most cases, users need only make changes on mocap.local. On your own personal computer, ensure you have the following
+software:
+
+1. **ROS (melodic)**: For interface with mocap.local and CC during flight.
+2. **QGroundControl**: For setup of flight controller.
+3. **Terminator**: It makes the multitude of terminals that you will be needing a little tidier.
+
+Within ROS, you might need certain packages and associated software (e.g the vrpn client, rviz, gazebo)
+and so a full ROS-desktop install is adviced. For reference, you could use the following script:
+::
+
+    sudo apt-get install ros-melodic-mavros ros-melodic-mavros-extras ros-melodic-vrpn-client-ros
+    sudo wget https://raw.githubusercontent.com/mavlink/mavros/master/mavros/scripts/install_geographiclib_datasets.sh
+    ./install_geographiclib_datasets.sh
+
+
+Companion Computer Setup
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You will need to run a similar setup on the companion computer (CC). For simplicity, we have prepared disk images that can be cloned
+via Clonezilla straight onto new boards. They currently come in three different flavours:
+
+1. Odroid
+2. Upboard
+3. TX2
+
+Borrow the relevant USB stick from the lab.
+
+Once you have flashed the relevant image and AFTER you have setup the flight controller (see next section) you will need to configure
+one of the board's serial ports for communication with the flight controller.
+
+.. TODO: Generate quickstart script and usb sticks
+
+Flight Controller Setup 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+**Current Firmware Version: PX4 v1.9**
```

Comment:
are you sure everything is on 1.9?

Reply:
I think the current version of quads use v1.7.3 (or v1.7.4)

Reply:
hmmm... this might be a little problematic. the px4 wiki only goes back to v1.8. I have a quad with me that runs on v1.9 (wasn't on v1.10 as back then... their documentation was still work-in-progress). I believe switching to v1.9 would lose only the direct motor control from zijian's stuff?